encord/orm/dataset.py

```diff
@@ -921,3 +921,36 @@ class ImagesDataFetchOptions:
     Whether to fetch signed urls for each individual image. Only set this to true if you need to download the 
     images.
     """
+
+
+class LongPollingStatus(str, Enum):
+    PENDING = "PENDING"
+    DONE = "DONE"
+    ERROR = "ERROR"
+
+
+@dataclasses.dataclass(frozen=True)
+class DatasetDataLongPolling(Formatter):
+    status: LongPollingStatus
+    data_hashes_with_titles: List[DatasetDataInfo]
+    errors: List[str]
```

Comment:
What exactly is reported for errors?

Reply:
[error_csv.csv](https://github.com/encord-team/encord-client-python/files/11049902/error_csv.csv)

This is error CSV from FE, `errors: List[str]` will correspond to `error_message` column in this CSV (exactly)